process/block/metablock_test.go

```diff
@@ -266,6 +294,78 @@ func TestNewMetaProcessor_NilForkDetectorShouldErr(t *testing.T) {
 	assert.Nil(t, be)
 }
 
+func TestNewMetaProcessor_NilEpochStartDataCreatorShouldErr(t *testing.T) {
+	t.Parallel()
+
+	coreComponents, dataComponents, bootstrapComponents, statusComponents := createMockComponentHolders()
+	arguments := createMockMetaArguments(coreComponents, dataComponents, bootstrapComponents, statusComponents)
```

Comment:
could refactor to `createMockMetaArguments(createMockComponentHolders())` in all tests where you don't change a component holder